functions/src/project/update-repositories.ts

```diff
@@ -35,8 +36,8 @@ export const onUpdateProjectRepositories: any = firestore
         const repoData: DocumentData = (await FirebaseAdmin.firestore().collection('repositories').doc(repositoryUid).get()).data();
 
         if (Array.isArray(repoData.projects)) {
-          const foundIndex: number = repoData.projects.findIndex((element: string) => element === context.params.projectUid);
-          if (foundIndex === -1) {
+          const project: ProjectModel = repoData.projects.find((element: string) => element === context.params.projectUid);
+          if (project) {
```

Comment:
Thank you for making these changes, I am going to revert this file, because I think there are more es6 changes needed in the functions and it would be better if they are all in a single commit with no new features - my fault sorry